src/routes.jsx

```diff
@@ -7,6 +7,7 @@ import JsTeam from './views/JsTeam';
 import NimbledroidGraphPage from './views/NimbledroidGraph';
 import Quantum from './quantum/index';
 import TP6 from './views/QuantumTP6/index';
+import TP6M from './views/QuantumTP6m/index';
```

Comment:
No need to add `/index` at the end: `import TP6M from './views/QuantumTP6m';`. The index file is the default import.